explanation/anbox-security.md

```diff
@@ -80,6 +80,21 @@ It is possible to turn off this update mechanism by setting `container.security_
 
 For security reasons, always keep your systems up-to-date at all times. To ensure this, snaps update automatically, and the snap daemon is by default configured to check for updates four times a day.
 
+## Data security
+
+We avoid storing user data as much as possible and don't provide any data encryption. The following table helps you understand how data related to you or provided by you is used within Anbox Cloud by various components.
```

Comment:
Should we add a small paragraph to make it very clear that it is the user's responsibility to secure this data?